src/tests/utils/fixtures/endpoint_request_fixture.py

```diff
@@ -0,0 +1,16 @@
+
+
+class Request:
+    def __init__(self, url_rule=None, method=None, headers=None):
+        self.url_rule = url_rule
+        self.method = method
+        self.headers = headers
+
+
+def create_request(url, method, access_token):
+
+    request = Request(
+        url_rule=url, method=method, headers={"access_token": access_token}
```

Comment:
Use the authorization header: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Authorization

Reply:
Done.